src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -2,30 +2,139 @@ import {ResultsViewPageStore} from "../resultsView/ResultsViewPageStore";
 import {SampleGroup} from "./GroupComparisonUtils";
 import {remoteData} from "../../shared/api/remoteData";
 import ListIndexedMap from "../../shared/lib/ListIndexedMap";
-import {Sample, SampleIdentifier} from "../../shared/api/generated/CBioPortalAPI";
+import {
+    MolecularProfile,
+    MolecularProfileFilter,
+    Sample, SampleFilter,
+    SampleIdentifier
+} from "../../shared/api/generated/CBioPortalAPI";
+import {action, computed, observable} from "mobx";
+import client from "../../shared/api/cbioportalClientInstance";
+import _ from "lodash";
+import {
+    pickCopyNumberEnrichmentProfiles, pickMRNAEnrichmentProfiles,
+    pickMutationEnrichmentProfiles, pickProteinEnrichmentProfiles
+} from "../resultsView/enrichments/EnrichmentsUtil";
+import MobxPromiseCache from "../../shared/lib/MobxPromiseCache";
+import {AlterationEnrichment} from "../../shared/api/generated/CBioPortalAPIInternal";
+import {makeEnrichmentDataPromise} from "../resultsView/ResultsViewPageStoreUtils";
+import internalClient from "../../shared/api/cbioportalInternalClientInstance";
 
 export default class GroupComparisonStore {
     constructor(
-        private resultsViewPageStore:ResultsViewPageStore
+        private __TEMP_SAMPLEGROUPSPROMISE:any
```

Comment:
Any specific reason for not passing `resultsViewPageStore`

Reply:
@alisman and I spoke and were saying that we don't want GroupComparisonStore to depend on ResultsViewPageStore anymore

Reply:
ok